### DISCUSSION TITLE
Problem with Lighthouse Deck

### DISCUSSION THREAD

**Discussion by djyamunaq** [bitcraze/discussions#764] [Q&A]
Hi,

I'm having now a problem with lighthouse deck. I connected it to the crazyflie 1.1 and updated the firmware through the ios app. When I turn on the crazyflie after updating, the red led of the lighthouse deck doesn't turn off and I can't use the V1 base stations. Here goes the console log:

---SYS: Crazyflie Bolt 1.1 is up and running!
SYS: Build 0:7d40b5411085 (2023.06) 
CLEANSYS: I am 0x303032333331470A0027003D and I have 1024KB of flash!
CFGBLK: v1, verification [OK]
DECK_CORE: 1 deck(s) found
DECK_CORE: Calling INIT on driver bcLighthouse4 for deck 0
IMU: BMI088: Using SPI interface.
IMU: BMI088 Gyro connection [OK].
IMU: BMI088 Accel connection [OK]
IMU: BMP388 I2C connection [OK]
ESTIMATOR: Using Kalman (2) estimator
CONTROLLER: Using PID (1) controller
MTR-DRV: Using brushless motor driver
SYS: About to run tests in system.c.
SYS: NRF51 version: 2023.02
EEPROM: I2C connection [OK].
STORAGE: Storage check [OK].
IMU: BMI088 gyro self-test [OK]
SYS: Self test passed!STAB: Wait for sensor calibration...
SYS: Free heap: 19184 bytes
LHFL: Lighthouse bootloader version: 2
STAB: Starting stabilizer loopSUP: Can not fly
ESTKALMAN: State out of bounds, resetting
LHFL: Bitstream CRC32: 952E7A01 [FAIL]
LHFL: The deck bitstream does not match the required bitstream.
LHFL: We require lighthouse bitstream of size 104093 and CRC32 112BC794.
LHFL: Leaving the deck in bootloader mode ...
LH: FPGA not booted. Lighthouse disabled!
ESTKALMAN: State out of bounds, resetting _(repeats indefinitely)_

Searching I've found that it might be that the firmware is not properly updated for the lighthouse deck. It looks like it's ok for the crazyflie 1.1 itself, and I have no problem connecting without the deck. 

Does someone know how I can deal with it? Can I update the firmware for the deck through usb, for example?

Thank's!

ps: I can only try to update the firmware through the iOS app, I don't have now a Crazyradio PA and can't really get one soon, as we are in a budget in this project.

**Comment by krichardsson** [comment]
Hi,
it is unfortunately only possible to update the lighthouse deck through the Crazyradio, sorry!

**Comment by djyamunaq** [comment]
Hi,

Remembering our previous conversation in which you recommended me the
products:

Hi,

Ah OK, if you already have base stations, you would only need a Crazyflie
(or Crazyflie Bolt) and a lighthouse deck. If you don't want to connect to
the Crazyflie using a USB cable you will also need a Crazyradio.

https://store.bitcraze.io/collections/kits/products/crazyflie-2-1or
https://store.bitcraze.io/collections/kits/products/crazyflie-bolt-1-1
https://store.bitcraze.io/collections/decks/products/lighthouse-positioning-deck
https://store.bitcraze.io/collections/kits/products/crazyradio-2-0

I would need to ask you to be more clear and careful since now I bought two
expensive and not useful products.

Regards,
Denis Yamunaque

**Comment by krichardsson** [comment]
Sorry about that, my bad!
It's hard to think about all the ins and outs, sometimes we get it wrong :-(

Ill send you a Crazyradio to fix the problem, just drop us an email on support@bitcraze.io with your order number

/ Kristoffer

**Comment by djyamunaq** [comment]
Hi, Kristoffer,

No worries. I'll talk to the part responsible for the orders in the lab
and get back to you as soon as possible. Thank you for the support.

Regards,
Denis Yamunaque

**Comment by djyamunaq** [comment]
Hi, Kristoffer,

The order number passed by the responsible was *#6892*.

Please, let me know if you need further information.

Regards,
Denis Yamunaque

**Reply by krichardsson** [reply]
That should be fine
Thanks!
